index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Somarib Portfolio</title>
+
+    <link rel="stylesheet" href="stylesheet.css">
+  </head>
+  <body>
+    <div class="maincont">
+
+      <div class="app-bar">
+        <div class="upper-bar">
```

Comment:
- [ ] Kindly change the `div` with a class of `app-bar` to a `header` tag 🙏 . This would make your code semantic and SEO friendly